base_usability/fix-odoo-base-multi-company-partners-not-shared.diff

```diff
@@ -0,0 +1,18 @@
+diff --git a/odoo/addons/base/res/res_users.py b/odoo/addons/base/res/res_users.py
+index c621ca4..d3880fe 100644
+--- a/odoo/addons/base/res/res_users.py
++++ b/odoo/addons/base/res/res_users.py
+@@ -334,7 +334,12 @@ class Users(models.Model):
+         user = super(Users, self).create(vals)
```

Comment:
The code here has been changed in v12 to create_multi.
https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/res_users.py#L423-L429

But i don't use this patch, then not a problem for me

Reply:
I updated this patch in my last commit anyway